### ISSUE TITLE
[enterprise-4.6] Issue in file service_mesh/v2x/upgrading-ossm.adoc

### ISSUE THREAD

**Issue by pamoedom** [openshift/openshift-docs#27515]
### Which section(s) is the issue in?

https://docs.openshift.com/container-platform/4.6/service_mesh/v2x/upgrading-ossm.html#ossm-upgrading-config-tracing_upgrading-ossm

### What needs fixing?

The Jaeger example configuration is incorrect regarding the storage variable and this avoids the proper production Elasticsearch mode to be enabled (see the extra "type" variable).

Current:

~~~
spec:
  addons:
    jaeger:
      name: jaeger
      install:
        storage: Memory # or Elasticsearch
        memory: {} # details specific to memory storage
        elasticsearch: {} # details specific to elasticsearch storage
  runtime:
    components:
      tracing.jaeger: {} # general jaeger specific runtime configuration
      tracing.jaeger.elasticsearch: {} # runtime configuration for jaeger elasticsearch deployment
~~~

Desired:

~~~
spec:
  addons:
    jaeger:
      name: jaeger
      install:
        storage:
          type: Memory # or Elasticsearch
        memory: {} # details specific to memory storage
        elasticsearch: {} # details specific to elasticsearch storage
  runtime:
    components:
      tracing.jaeger: {} # general jaeger specific runtime configuration
      tracing.jaeger.elasticsearch: {} # runtime configuration for jaeger elasticsearch deployment
~~~

Best Regards.

**Comment by pamoedom** [comment]
Please see also related https://github.com/openshift/openshift-docs/issues/27514

**Comment by JStickler** [comment]
Closed via #27568
